database/sqlite/sqlite_aclk.c

```diff
@@ -207,7 +189,7 @@ int aclk_start_sync_thread(void *data, int argc, char **argv, char **column)
 
 void sql_aclk_sync_init(void)
 {
-#ifdef ACLK_NEWARCH_DEVMODE
+#if ENABLE_NEW_CLOUD_PROTOCOL
```

Comment:
Hi Stelio! Is there a reason this is `#if` instead of `#ifdef` ?

Reply:
I will change